File: Example/FlexLib/FrameVC.m
/**
 * Copyright (c) 2017-present, zhenglibao, Inc.
 * email: 798393829@qq.com
 * All rights reserved.
 *
 * This source code is licensed under the MIT-style license found in the
 * LICENSE file in the root directory of this source tree.
 */

#import "FrameVC.h"

@interface FrameVC ()
@property(nonatomic,strong) FlexFrameView* frameView;
@end

@implementation FrameVC

- (void)viewDidLoad {
    [super viewDidLoad];
    
    FlexFrameView* view = [[FlexFrameView alloc]initWithFlex:@"FrameVC" Frame:CGRectZero Owner:self];
    self.frameView = view;
    self.view = view;
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    
    
}
- (void)dealloc
{
    NSLog(@"FrameVC dealloc");
}
@end

File: Example_swift/FlexSwiftDemo/FrameVC.swift
//
//  FrameVC.swift
//  FlexSwiftDemo
//
//  Created by zhenglibao on 2018/2/5.
//  Copyright © 2018年 wbg. All rights reserved.
//

import UIKit
import FlexLib

class FrameVC: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        self.view = FlexFrameView.init(flex: "FrameVC", frame: CGRect.zero, owner: self)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

}
